build/Linux+BSD/portable/RecipeDebian

```diff
@@ -153,6 +153,7 @@ package() {
   appimage="$(echo "$appdir" | sed 's|\.AppDir$|.AppImage|')"
 
   cd AppImageKit-5/AppImageAssistant.AppDir
+  chmod u+x /bin/QtWebEngineProcess 
```

Comment:
not `chmod u+x bin/QtWebEngineProcess ` i.e. without the leading slash and as sich relative to `AppImageKit-5/AppImageAssistant.AppDir`?